assets/sass/okta-sign-in.scss

```diff
@@ -7,6 +7,21 @@
 @import 'common/enduser/responsive-variables';
 @import 'common/enduser/helpers';
 @import 'container';
+@import 'modules/okta-footer';
```

Comment:
Not entirely sure, but should this one be scoped to the widget only, like it was before?

Reply:
oh got it, that answers my question in a comment I just made. Thanks

Reply:
mind filing a jira for later?

Reply:
Done :) https://oktainc.atlassian.net/browse/OKTA-141201